tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/VulnerabilityBatchTests.SqlInjection.cs

```diff
@@ -129,6 +129,46 @@ public void GivenASqlQueryWithStringLiteral_WhenSerialized_JsonIsRedacted(string
             _defaultScrubber);
     }
 
+    [Theory]
+    [InlineData("john")]
+    [InlineData("username with 🌉 surrogate")]
+    public void GivenAMySqlQueryWithStringLiteral_WhenSerialized_JsonIsRedacted(string literal)
```

Comment:
There are a couple of things that scare me from this approach:

- Adding a new test case to the `yaml` file should not force you to write any `cs` code
- Data should not be duplicated in the `yaml` and `cs` files to prevent issues if the former changes (for instance the evidence and the asserted json)

I think the best course of action is to read the whole yaml file and then use `[MemberData]` or `[ClassData]` to provide an `IEnumerable` that can be used in a parameterized test (check [this article](https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/) from @andrewlock)

Reply:
Those are the legacy ones. This PR is to include the and process the yml itself. Look at the `VulnerabilityBatchTests.Bundle.cs` file

Reply:
Implemented in 0467ff395482402d4748222ba99e5861cbf87e66